src/LuaTransformer.ts

```diff
@@ -2812,7 +2821,7 @@ export class LuaTransformer {
                 return this.transformBinaryBitLibOperation(node, left, right, operator, "bit");
 
             default:
-                if (operator === tstl.SyntaxKind.BitwiseArithmeticRightShift) {
+                if (operator === tstl.SyntaxKind.BitwiseRightShiftOperator) {
                     throw TSTLErrors.UnsupportedForTarget("Bitwise >>> operator", this.options.luaTarget, node);
```

Comment:
```suggestion
                     throw TSTLErrors.UnsupportedForTarget("Bitwise >> operator", this.options.luaTarget, node);
```

Reply:
Incorrect, >> is the arithmetic right shift (which is supported), >>> is the logical right shift (not supported).

Reply:
`>>>` in JS is `>>` in Lua
`>>` in JS has no direct replacement in Lua

Reply:
no  `>>` in js is arithemtic which is `>>>` in lua

Reply:
There is no `>>>` in lua 5.3

Reply:
>> is arithmetic in JS, 5.3 ONLY has arithmetic bit shift operators, so we accidentally got it right the first time for 5.3

Reply:
No 5.3 only has logical shifts at least according to the docs
> Both right and left shifts fill the vacant bits with zeros.
https://www.lua.org/manual/5.3/manual.html

* so `>>>` should be transpiled to `>>`
* `<<` to `<<`
* `>>` is not supported

not sure what we currently do tbh
what a confusing issue